openshiftcontrolplane/v1/types.go

```diff
@@ -46,8 +46,38 @@ type OpenShiftAPIServerConfig struct {
 
 	// TODO this needs to be removed.
 	APIServerArguments map[string][]string `json:"apiServerArguments"`
+
+	// apiServers holds information about enabled/disabled API servers
+	ApiServers ApiServers `json:"apiServers"`
```

Comment:
Nit, should this not be APIServers instead of ApiServers? I'd have expected all uses in names/types here to be APIServers

Reply:
Done